specfile/specfile.py

```diff
@@ -0,0 +1,106 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import contextlib
+import types
+from pathlib import Path
+from typing import Iterator, List, Optional, Tuple, Type, Union
+
+from specfile.rpm import RPM, Macros
+from specfile.sections import Sections
+from specfile.tags import Tags
+
+
+class Specfile:
+    """
+    Class representing spec file.
```

Comment:
Nitpick: missing article for `spec file` (slighly lower `the` is used but `a` is probably more suitable here)

Reply:
I was trying to be consistent with other `packit` projects, namely `ogr`, there are no articles in class docstrings there. But it probably wasn't the best idea, I'll add the missing articles everywhere.